src/PHPStan/Analysis/ErrorTransformer.php

```diff
@@ -15,7 +15,7 @@ public function transform(array $errors): array
     {
         return array_map(
             function ($error) {
-                if (! str_contains($error->message, '$')) {
```

Comment:
There is both `if (! $foo)` and `if (!$foo)` in the codebase.
I'm ok with both, but it should be consistent.